harness/go/history/fetcher.go

```diff
@@ -98,7 +100,7 @@ func (f *Fetcher) GetExecutions(ctx context.Context) ([]*workflow.WorkflowExecut
 			Namespace:       f.Namespace,
 			MaximumPageSize: 1000,
 			NextPageToken:   nextPageToken,
-			StartTimeFilter: &filter.StartTimeFilter{EarliestTime: &earliest},
+			StartTimeFilter: &filter.StartTimeFilter{EarliestTime: timestamppb.New(earliest)},
```

Comment:
(More a general gogoproto migration question) Just to confirm this would be a visible change to the API for users correct? and we are OK with  that?

Reply:
To the wire format or to the Go API? The wire format will remain unchanged

Reply:
I meant an API change

Reply:
Ah, yes. It will be and there's no way around it